pkg/controllers/provisioning/provisioner.go

```diff
@@ -234,6 +234,10 @@ func (p *Provisioner) NewScheduler(ctx context.Context, pods []*v1.Pod, stateNod
 		if err != nil {
 			return nil, fmt.Errorf("getting instance types, %w", err)
 		}
+		if len(instanceTypeOptions) == 0 {
+			logging.FromContext(ctx).With("provisioner", nodePool.Name).Info("excluding from scheduling, no instance types exist")
```

Comment:
```suggestion
			logging.FromContext(ctx).With("provisioner", nodePool.Name).Info("skipping provisioner, no resolved instance types found")
```

Align the two log messages for provisioning/deprovisioning?